docs/Developers/Newcomers_Guide.rst

```diff
@@ -207,10 +209,34 @@ The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of what
 it's like to work with us and what the workflow is like.
 
+The coala community creates a constant supply of ``newcomer`` issues available
+for people like you who are joining the organisation, so that you can learn
+how to contribute according to the coala organisation development guides,
+while doing a real task at the same time.
+
+coala does **not** follow the GitHub "flow" workflow, so be sure to read
+our guides carefully. You will need to use the ``git`` command line.
+During the newcomer process, we will help you conquer ``git``, so that you
+can become a part of our super efficient developer team.
+
+If you do not follow the coala workflow, our bots will get grumpy, and
+members of our community will try to help you in every way possible to
+get your Pull Request merged, the coala way.
+
+Your Pull Request will **not** be accepted until it complies with coala
+standards, even if your solution is amazingly correct.  If you ignore the
+bots or our developers, or you close your Pull Request, we will have
+no option except to conclude that you have abandoned the coala newcomer
+process, and your amazing solution **will** be discarded.  Luckily, all
```

Comment:
I think it will be good to add here that making a new PR for the same issue isn't recommended, or closing a PR to make a new one due to troubles with conflicts or bot complaints.  Instead we should nudge them to seek help from the manual or the community if they are having troubles to reach a resolution.